workflows/testing/test_workflow/integ_test.py

```diff
@@ -0,0 +1,13 @@
+import os
+
+class IntegTestSuite:
+    def __init__(self, name, repo):
+        self._name = name
+        self._repo = repo
+
+    def execute(self, cluster):
+        script = self._repo.dir() + "/integtest.sh"
+        if (os.path.exists(script)):
+            repo.execute(f'sh integtest.sh -b {cluster.endpoint()} -e {cluster.port()}')
+        else:
+            print(f'{script} does not exist. Skipping integ tests for {self._name}')
```

Comment:
This seems like an exceptional state, shouldn't all repositories include integration tests?  I'd recommend throwing with a nasty error so they have to be dealt with

Reply:
Eventually they should all include integration tests. We're not there yet, however.

Reply:
I tend to lend on not included in components into bundling until they meet a basic compliance level.  Forces earlier investment via components and makes our 'management' of onboarding new components simpler.

Otherwise we have to scan the logs for this error message that means we have an untested component.

/2cents

Reply:
Directionally, I agree. Since this is a prototype I'm happy to worry about that later.